src/__tests__/scenes/RequestScene.test.tsx

```diff
@@ -12,6 +12,7 @@ describe('Request', () => {
 
     const actual = renderer.render(
       <RequestSceneComponent
+        account={null as any}
```

Comment:
We typically do `const fakeAccount: any = {}` at the top of the file, and then pass `fakeAccount` to the prop. Just a style thing.

Reply:
I just copied the style of the file, and the wallet object was done this way. Confirm that you find enought priority for me to change and I'll go for it.

Reply:
Notes for next time, I guess.